backend/routes/post.py

```
@@ -18,6 +18,7 @@
 from core.connection_crud import get_connections
 from crud.notification import create_notification
 from AI.moderation import moderate_text
+from services.services import *
```

Ruff (0.8.2)

`from services.services import *` used; unable to detect undefined names

(F403)